mod/posts/set/type/post.rb

```diff
@@ -51,7 +46,7 @@ def tab_list
     render_title_link
   end
 
-  view :bar_right, cache: :never do
```

Comment:
so what's in bar_right now?

Reply:
empty

Reply:
why did you decide to do it that way?

Reply:
Because I wanted to make use the middle section. I didn't like that there are 3 badges squeezed to the far right, which felt weird.

Reply:
I am not trying pose any strong pattern on where the badges should go in the bar and boxes. We should add content based on the context, importance and look/feel. We should be flexible with that. The default is defined, left most has a picture, name and subtitle, far right has icons to the page and to expand in place.